packages/contracts/contracts/utils/Ratio.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+
```

Comment:
I think by using `getRatio() returns (uint256)`, we would need to include another function in the contract whereas just using `RATIO_BASE` requires less storage.

I thought about adding `applyRation(value, total)`/ `toRatio(uint256 _numerator, uint256 _denominator)`. However, it would not have been used anywhere, so I didn't included it.

Reply:
constants don't require any storage at all. The compiler replaces the references in the code with the actual value during the compilation.

Reply:
Done in 4e2cc59df09a26184936ef9bf3bb5f131479eeb0